app/controllers/admin/personalities_controller.rb

```diff
@@ -1,11 +1,18 @@
 class Admin::PersonalitiesController < Admin::BaseController
-  before_action :set_personality, only: [:show, :edit, :update, :destroy]
+  before_action :set_personality, only: [
+    :show, :edit, :update, :destroy,
+    :change_role_to_guest,
+    :change_role_to_editor,
+    :change_role_to_secret,
+    :change_role_to_admin,
```

Comment:
__[RuboCop]__

Style/TrailingCommaInLiteral: Avoid comma after the last item of an array, unless each item is on its own line.


<!-- Comment by [SideCI](https://sideci.com) -->